### DISCUSSION TITLE
Getting a spam email notification, need help

### DISCUSSION THREAD

**Discussion by POP1912** [community/community#42845] [Discussions] [answered]
### Select Topic Area

General

### Body

I received an email from - "**evilAngel** <notifications@github.com>"  stating that I have been mentioned to some repo which I never heard of. And got a reply to that message with some offensive messages...
Can anyone help me in this

Below is the screen capture - 
![image](https://user-images.githubusercontent.com/48610563/209950564-2308bb99-d59f-4cf5-8b8a-0d77be436ee3.png)

**Comment by geckolinux** [comment]
https://github.com/community/community/discussions/42810

**Comment by Akash1134** [comment, marked as answer]
Hi @POP1912, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.

**Reply by POP1912** [reply]
Hi @Akash1134. I currently marked that mail as spam as a result all github notifications were treated as spam in Gmail. Please let me know if any action is required from my end. Thanks

**Reply by GottaLaff** [reply]
If this is anything like my issue, getting spammed by a dozen or more emails a day from 
whosumdotcom8[bot]  (only each one has a different number), then PLEASE let me know what to do. I also treat them as spam in gmail.

**Reply by GottaLaff** [reply]
<img width="1505" alt="Screenshot 2024-01-24 at 12 53 51 PM" src="https://github.com/community/community/assets/105238421/91ebde12-9469-4998-a615-9b53492d8e99">

Please make this stop. I keep unsubscribing. I leave "Stop spamming my email" comments on each one. No change.

**Comment by Ankush11903** [comment]
![Screenshot (623)](https://user-images.githubusercontent.com/91266368/210373864-7fe9d39f-885f-4cec-b3ac-a9d0aa130495.png)
You have to turned it off for now until some time has passed and the bots stop being an issue.

**Reply by POP1912** [reply]
Thank you  Ankush

**Reply by Ankush11903** [reply]
Welcome @POP1912 .
If it helps you please mark it as answered :).

**Reply by GottaLaff** [reply]
I've turned off all notifications. I'm STILL being spammed daily in my email. I've deleted the most recent, but here you go.
<img width="1505" alt="Screenshot 2024-01-24 at 12 53 51 PM" src="https://github.com/community/community/assets/105238421/694db76d-7a0f-4e62-9534-6c99062eada1">

**Comment by GottaLaff** [comment]
I get a dozen emails a day now, starting a couple of weeks ago, from, for example: "[whosumdotcom/srchmdonauthorposts] Updating mdon/authorlatest/@gottalaff@mastodon.social.json"  How do I stop these? I keep unsubscribing, nothing happens. I don't even understand what this is. But it has to stop.

**Comment by shakna-israel** [comment]
More spam: https://github.com/alehander92/Airtight/issues/1#issuecomment-1953321293

**Comment by teneko** [comment]
New mechanic is being abused to spam by mentioning people in PRs and get control of your account to spread more fraud and spam: https://github.com/orgs/community/discussions/109171

**Reply by ettaboyle** [reply]
@teneko thank you for that link, I've reported the account for phishing.

**Comment by ettaboyle** [comment]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by GottaLaff** [comment]
Thank you.

I managed to find a way to unsubscribe and ignore. I don't know my way
around this site, nor understand why I'm part of it, but I did solve the
problem at my end only.

Thanks again.
